platform/stm32f4xx/export/platform/i2c_bus.hpp

```diff
@@ -496,6 +510,40 @@ constexpr auto i2c_bus< i2c_config >::pick_rcc_fn()
 }
 
 template< class i2c_config >
+void i2c_bus< i2c_config >::receive_byte(size_t count)
```

Comment:
Let's rename it to receive_byte**s** to reflect real nature of the method
Receive in poll mode also can be abstracted by using this method:
https://github.com/forGGe/theCore/pull/18/commits/4aa74f1f01d2dc34c23eaa136c1f35a395ab7ca0#diff-759032371b07cd467947dc127cbffcb5R383